### ISSUE TITLE
Profile : Uploading high resolution avatar image fails 

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#13962]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Get attached image on your iPhone
2. Navigate to settings/profile and upload it as avatar
3. Crop screen won't appear

## Expected Result
Crop screen should appear, or error message should be shown if image is too large.

## Actual Result
Picked image silently fails

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?

- [ ] iOS / Safari

**Version Number:** 1.2.47-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/210434876-54add13c-0679-42ec-aee2-59e6ee68b1dc.mp4

**Expensify/Expensify Issue URL:**
**Issue reported by:** @azimgd 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1672734548804589

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by jasperhuangg** [comment]
Ahhh jeez cc @Beamanator since we've had some fun with these avatar before :D

@kavimuru can you please also provide the image that you used to cause it to fail? How large was it?

**Comment by Beamanator** [comment]
Sheesh 🙃  this looks like fun - FYI it looks like @azimgd posted an example image that is causing failures here: https://expensify.slack.com/archives/C049HHMV9SM/p1672734756858749?thread_ts=1672734548.804589&cid=C049HHMV9SM

**Comment by francoisl** [comment]
> some images that have a ton of images also have a high pixel density or something

I'm not following, can you rephrase please?

Anyway, I just tried on `main` on mWeb Safari and can't reproduce, I'll try with a different "large" image.

https://user-images.githubusercontent.com/2229301/211639243-390452bb-c392-4b29-b24d-280c999012b2.mov

**Comment by Beamanator** [comment]
@francoisl I mean to say:

> some images that have a ton of pixels also have a high pixel density or something

I think it's called DPI - "Dots per inch" (according to google). My main point was that if I remember correctly, this image you're testing with has a ton of pixels but also a very high pixel density, which I thoughttt was why it could be a small file size though it has a ton of pixels... Anyway good luck trying to reproduce 👍

**Comment by francoisl** [comment]
Ah ok, got it now. I don't think DPI matters in this case, only the image dimensions should.

I tried with a bunch of other images over and below 4096x4096 and still can't reproduce the error, either I get the error message or the AvatarCropModal opens as expected.
Going to close, please reopen if you can reproduce and attach the exact image you were testing with.
